.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,21 +1,18 @@
 ---
 name: Bug report
-about: Create a report to correct failures and improve our code
+about: github issue template for bugs and code improvement suggestions
 title: ''
 labels: ''
 assignees: ''
 ---
-**Describe the bug**
```

Comment:
i thins this should stay

Reply:
> i thins this should stay

OK, that's what Daniel said.  The problem I am trying to solve, getting more users to provide the info we're asking for , may be largely out of my control.